StRoot/macros/calib/Calib_SC_GL.C

```diff
@@ -2036,18 +2041,23 @@ int FitWithOutlierRemoval(int debug) {
     if (eplus!=0.0 && eminus!=0.0) fitParErrs[k] = 0.5*(eplus-eminus);
     else if (eplus!=0.0 || eminus!=0.0) fitParErrs[k] = eparab;
     printf("%s\t:\t%g\t+/- %g\n",parName[k],fitPars[k],fitParErrs[k]);
-    for (i=0;i<3;i++) if (i!=k) minuit->ReleaseParameter(i);
+    for (i=0;i<npar;i++) if (i!=k) minuit->ReleaseParameter(i);
   }
 
   // One more determination of variance using final fit,
-  //   without outlier removal
+  //   with and without outlier removal
   memcpy(outliers1,outliers,nBytes);
   memset(outliers,0,nBytes);
   (*(minuit->GetFCN()))(k,&temp1,temp2,fitPars,status);
-  VAR_all = 0;
-  for (i=0; i<nMeasures; i++) VAR_all += devs[i]*devs[i];
-  VAR_all = TMath::Sqrt(VAR_all/((Double_t) nMeasures));
   memcpy(outliers,outliers1,nBytes);
+  VAR = 0;
+  VAR_all = 0;
+  for (i=0; i<nMeasures; i++) {
+    if (!outliers[i]) VAR += devs[i]*devs[i];
+    VAR_all += devs[i]*devs[i];
+  }
+  VAR = TMath::Sqrt(VAR/((Double_t) nMeasuresI-1));
+  VAR_all = TMath::Sqrt(VAR_all/((Double_t) nMeasures-1));
```

Comment:
In arithmetic expressions involving doubles integer types are implicitly converted to double.  So, the cast to `(Double_t)` in `VAR_all/((Double_t) nMeasures-1)` is not really needed.

Reply:
Thanks for the point, Dmitri. The casting makes clear what is happening, and is not detrimental, so I would rather leave it and move on.